### PR TITLE
Add notifications

### PR DESCRIPTION
**Pull request by jonchan51**
and fix start game, it did not match what was happening on the backend which made quiting and starting another duet game quirky

I added notifications to all the places I noted last time, not sure if there are more, feel free to add on.

### CODE REVIEW

File: src/components/Duet/DuetRoomHeader.tsx [hunk 1]
```diff
@@ -108,10 +109,13 @@ const DuetRoomHeader: React.FC<Props> = ({
...
               className={classes.link}
-              onClick={
-                async () => navigator.clipboard.writeText(window.location.href)
-                // TODO add a notification
-              }
+              onClick={async () => {
+                navigator.clipboard.writeText(window.location.href);
+                displayNotification({
+                  message: 'Copied to clipboard',
+                  severity: 'info',
```
Comment (Na-Nazhou) [inline src/components/Duet/DuetRoomHeader.tsx:116]: will 'success' be better?

File: src/utils/socket.ts [hunk 2]
```diff
@@ -50,43 +55,116 @@ export function addListeners(
...
+          severity: 'info',
+        };
+      } else if (
+        !!prevFriend &&
+        !!curFriend &&
+        !prevFriend.ready &&
+        curFriend.ready
+      ) {
+        message = {
+          message: 'Your partner has readied',
```
Comment (Na-Nazhou) [inline src/utils/socket.ts:143]: Can we change to 'Your partner is ready now'

hmm, idk 'has readies' sounds weird

Comment (jonchan51) [reply src/utils/socket.ts:143]: Yeah i was trying to match the 'Your partner has unreadied', both feel really weird. I changed it to 
' Your partner is ready now'
and
'Your parter is not ready anymore'
Do they sound better?

Comment (Na-Nazhou) [reply src/utils/socket.ts]: yup sounds better

**Review by Na-Nazhou — APPROVED** [review]

**Pull request merged**